### DISCUSSION TITLE
Wi-Fi Beta

### DISCUSSION THREAD

**Discussion by erichelgeson** [BlueSCSI/BlueSCSI-v2#63] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/BlueSCSI/BlueSCSI-v2/releases/tag/wifi-beta'>Wi-Fi Beta</a>.</em>

**Comment by JoopIdema** [comment]
Just installed Wifi Beta on my LC475 running MacOS 8.1. Followed the instructions provided here. Works perfectly. Great job!
